vllm/model_executor/models/hunyuan_vision.py

```diff
@@ -847,7 +847,7 @@ def get_xdrope_input_positions(
                 .expand(-1, llm_grid_w + 1)
                 .reshape(-1)
             )
-            h_index[pos : pos + token_num] = 0
+            t_index[pos : pos + token_num] = 0
```

Comment:
`t` is the number of the input image. Therefore,
`t_index[pos : pos + token_num] = image_index`